system-addon/lib/TopSitesFeed.jsm

```diff
@@ -142,8 +141,8 @@ this.TopSitesFeed = class TopSitesFeed {
    * @param {bool} options.broadcast Should the update be broadcasted.
    */
   async refresh(options = {}) {
-    if (!this._tippyTopProvider.initialized) {
-      await this._tippyTopProvider.init();
+    if (!this.store.getState().TippyTop.initialized) {
```

Comment:
TippyTop initialization is going to take longer now on a cold cache (first run, for example). It has to load the default manifest we ship with and then fetch the remote one. So, maybe we want to mark as initialized after the defaults are loaded? The first new tab opened might only have the default set and we might end up requesting screenshots that aren't really needed. But we can show the stuff faster.